README.md

```diff
@@ -589,16 +591,16 @@ Nexus instance that hosts the permanent content: `cachito_nexus_hoster_username`
 
 The table below shows the supported package managers and their support level in Cachito.
 
-Feature                 | gomod | npm | pip |
----                     | ---   | --- | --- |
-Baseline                | ✓     | ✓   | ✓   |
-Content Manifest        | ✓     | ✓   | ✓   |
-Dependency Replacements | ✓     | x   | x   |
-Dev Dependencies        | ✓     | ✓   | ✓   |
-External Dependencies   | N/A   | ✓   | ✓   |
-Multiple Paths          | ✓     | ✓   | ✓   |
-Nested Dependencies     | ✓     | ✓   | x   |
-Offline Installations   | ✓     | x   | x   |
+Feature                 | gomod | npm | pip | yarn |
+---                     | ---   | --- | --- | ---  |
+Baseline                | ✓     | ✓   | ✓   | ✓    |
+Content Manifest        | ✓     | ✓   | ✓   | ✓    |
+Dependency Replacements | ✓     | x   | x   | x    |
+Dev Dependencies        | ✓     | ✓   | ✓   | x    |
```

Comment:
Should we count the ability to keep dev dependencies as ` ✓ `? I did not do it because yarn can not distinguish dev deps.

Reply:
Let's keep it as `x` until we add the ability to do that :+1:

Reply:
Speaking of dev dependencies, why is gomod marked as supported??

Reply:
For Go, we divide them into two categories: module level dependencies (whatever is needed to build the package) and package level dependencies (what was actually imported by the modules. This does not include all the packages provided by the imported modules).

There is a misconception here though, since they are not dev dependencies (dependencies required for testing would be included in both since we do not know what the user will actually build and ship).

We could keep it as checket with a `*`, `✓*`, or `✓¹`, and then add a note under the table to make this one clear. WDYT?

Reply:
Sounds good (doesn't need to be in this PR though).